patches/api/0429-Expose-hand-during-BlockCanBuildEvent.patch

```diff
@@ -0,0 +1,48 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: The456gamer <the456gamer@the456gamer.dev>
+Date: Mon, 21 Aug 2023 14:13:43 +0100
+Subject: [PATCH] Expose hand during BlockCanBuildEvent
+
+
+diff --git a/src/main/java/org/bukkit/event/block/BlockCanBuildEvent.java b/src/main/java/org/bukkit/event/block/BlockCanBuildEvent.java
+index a1350c0f74d445dca09eea6e10abac050bb06990..eaacc42e9a2bc07cbbbdeeacb0088a141c87798b 100644
+--- a/src/main/java/org/bukkit/event/block/BlockCanBuildEvent.java
++++ b/src/main/java/org/bukkit/event/block/BlockCanBuildEvent.java
+@@ -25,6 +25,7 @@ public class BlockCanBuildEvent extends BlockEvent {
+ 
+     protected BlockData blockData;
+     private final Player player;
++    private final org.bukkit.inventory.EquipmentSlot hand; // Paper - expose hand
+ 
+     @Deprecated
+     public BlockCanBuildEvent(@NotNull final Block block, @NotNull final BlockData type, final boolean canBuild) {
+@@ -37,12 +38,29 @@ public class BlockCanBuildEvent extends BlockEvent {
+      * @param type the id of the block to place
+      * @param canBuild whether we can build
+      */
++    @org.jetbrains.annotations.ApiStatus.Internal // Paper
```

Comment:
Can just mark this as deprecated instead. Actually, also mark this as `DoNotUse` to make sure we catch any that upstream might add. 

So replace `Internal` with `Deprecated` and `DoNotUse`. Just for this method tho, the other one is good.

Reply:
Just leave that one as is. No need to mention anything in the javadocs either